### PR TITLE
Update apple-mdm-push-certificate-get.md

### PR DESCRIPTION
**Pull request by msbemba**
Confirmed and added the 30 days grace period to the document.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1602

File: memdocs/intune/enrollment/apple-mdm-push-certificate-get.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ An Apple MDM Push certificate is required for Intune to manage iOS/iPadOS and ma
 
 For more information about enrollment options, see [Choose how to enroll iOS/iPadOS devices](ios-enroll.md).
 
-When a push certificate expires, you must renew it. When renewing, make sure to use the same Apple ID that you used when you first created the push certificate.
+When a push certificate expires, you must renew it within the grace period(30 days as of this writing). When renewing, make sure to use the same Apple ID that you used when you first created the push certificate.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/apple-mdm-push-certificate-get.md:46]:
```suggestion
When a push certificate expires, you must renew it within the grace period (30 days as of this writing). When renewing, make sure to use the same Apple ID that you used when you first created the push certificate.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger20** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and merge this PR. Thanks

**Comment by PRMerger4** [comment]
@lenewsad : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/enrollment/apple-mdm-push-certificate-get.md [hunk 2]
```diff
@@ -35,15 +35,14 @@ ms.collection:
 
 [!INCLUDE [azure_portal](../includes/azure_portal.md)]
 
-An Apple MDM Push certificate is required for Intune to manage iOS/iPadOS and macOS devices. After you add the certificate to Intune, your users can enroll their devices using:
+Upload and renew your Apple MDM push certificates in Microsoft Intune. An Apple MDM Push certificate is required to manage iOS/iPadOS and macOS devices in Microsoft Intune, and enables devices to enroll via: 
 
-- The Company Portal app.
+- The Intune Company Portal app
+- Apple bulk enrollment methods, such as the Device Enrollment Program, Apple School Manager, and Apple Configurator
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/apple-mdm-push-certificate-get.md:41]:
```suggestion
- The Intune Company Portal app.
- Apple bulk enrollment methods, such as the Device Enrollment Program, Apple School Manager, and Apple Configurator.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, apologies, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Comment by tiburd** [comment]
@lenewsad 
This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/office-authoring-guide/office-docs-authoring-overview?branch=main). We reviewed and merged this PR but encourage you to use the private repo for future contributions to Microsoft Docs.
Thanks!

**Comment by lenewsad** [comment]
> @lenewsad This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/office-authoring-guide/office-docs-authoring-overview?branch=main). We reviewed and merged this PR but encourage you to use the private repo for future contributions to Microsoft Docs. Thanks!

FYI @yogkumgit @msbemba

**Comment by tiburd** [comment]
> > @lenewsad This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/office-authoring-guide/office-docs-authoring-overview?branch=main). We reviewed and merged this PR but encourage you to use the private repo for future contributions to Microsoft Docs. Thanks!
> 
> FYI @yogkumgit @msbemba

@lenewsad 
Hi, Laura,
We appreciate your intention in sharing the info with Yogesh and Sriraman. However, they don't have access to the private repos.
Thanks!

**Pull request merged**